### PR TITLE
Added Code of Conduct to the scroll.

### PR DESCRIPTION
**Pull request by joker2600**
@ryankanno @rbvea

I tried to fit the Code of Conduct into the scroll. Not sure if that works or if it is just too wordy.

### CODE REVIEW

**Comment by rbvea** [comment]
Yeah... we might have to create another page for this.  I'm apparently making a couple more pages (blog, etc) so I'll check to see if I could easily make that happen.  Thanks for this though!

**Comment by sthapa** [comment]
Closing old PR
